skxray/fitting/xrf_model.py

```diff
@@ -1059,15 +1076,12 @@ def linear_spectrum_fitting(spectrum, params, elemental_lines=None,
     # ratio to transfer energy value back to channel value
     approx_ratio = 100
```

Comment:
The default transfer from channel to energy is energy = 0.01*channel number. So to transfer back is just 100.